auth-web/src/assets/scss/actions.scss

```diff
@@ -1,3 +1,5 @@
+@import 'theme.scss';
```

Comment:
I noticed some parts you imported Theme as well as actions below

Reply:
removed, don't actually need this import